shared/signup/routes.tsx

```diff
@@ -2,6 +2,7 @@ export const newRoutes = {
   signupEnterDevicename: {getScreen: () => require('./device-name/container').default, upgraded: true},
   signupEnterEmail: {getScreen: () => require('./email/container').default, upgraded: true},
   signupEnterUsername: {getScreen: () => require('./username/container').default, upgraded: true},
+  signupSendFeedback: {getScreen: () => require('./feedback/container').default, upgraded: true},
```

Comment:
this should be a modal route

Reply:
lmk if the current impl is ok, i had to add a new prop to signupscreen to keep the styling consistent